### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by jmburgues**
Proposed PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Description :pencil:
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```
Comment (agustinb4) [inline .github/pull_request_template.md:5]: we could remove

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Link to ticket :link:
+
+Please include a link to Jira Ticket
```
Comment (agustinb4) [inline .github/pull_request_template.md:9]: Replace with the template: https://entropyteam.atlassian.net/browse/EN-

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Checklist: :white_check_mark:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (agustinb4) [inline .github/pull_request_template.md:33]: could we remove this section?

**Review by agustinb4 — APPROVED** [review]
Good idea :100:

**Comment by jmburgues** [comment]
@agustinb4 please verify and merge if looks good to you.

**Pull request merged**